projects/dsp-ui/src/lib/search/expert-search/expert-search.component.spec.ts

```diff
@@ -0,0 +1,134 @@
+import { Component, DebugElement, OnInit, ViewChild } from '@angular/core';
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+import { FormsModule, ReactiveFormsModule } from '@angular/forms';
+import { MatFormFieldModule } from '@angular/material/form-field';
+import { MatInputModule } from '@angular/material/input';
+import { By } from '@angular/platform-browser';
+import { BrowserAnimationsModule } from '@angular/platform-browser/animations';
+import { ActivatedRoute } from '@angular/router';
+import { RouterTestingModule } from '@angular/router/testing';
+import { KnoraApiConfig } from '@dasch-swiss/dsp-js';
+import { DspApiConfigToken } from '../../core/core.module';
+import { ExpertSearchComponent } from './expert-search.component';
+
+/**
+ * Test host component to simulate parent component.
+ */
+@Component({
+    template: `
+        <dsp-expert-search #expSearch></dsp-expert-search>`
```

Comment:
In the test host component you should register to the `@Output`s. Then you can trigger the emission and see if it had the desired effect. 

https://github.com/dasch-swiss/dsp-ui-lib/blob/b3b1fee1ecc97085f3bdb199c0352c36825b206a/projects/dsp-ui/src/lib/search/advanced-search/select-ontology/select-ontology.component.spec.ts#L14-L44

https://github.com/dasch-swiss/dsp-ui-lib/blob/b3b1fee1ecc97085f3bdb199c0352c36825b206a/projects/dsp-ui/src/lib/search/advanced-search/select-ontology/select-ontology.component.spec.ts#L117-L133

Reply:
done in 062ccea

Reply:
ok, great